aten/src/ATen/Dispatch.h

```diff
@@ -580,6 +606,38 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
 
 #define AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND3(                             \
     SCALARTYPE1, SCALARTYPE2, SCALARTYPE3, TYPE, NAME, ...)                 \
+  [&] {                                                                     \
```

Comment:
Why this is changed? I guess there is something wrong with the merge conflict. `AT_DISPATCH_ALL_TYPES_AND_C10_COMPLEX_AND3` has been removed and `AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND3` now uses `c10::complex`

Reply:
yeah fixed it